### ISSUE TITLE
Migration trying to create a new table when I rename a column

### ISSUE THREAD

**Issue by kurt-hardy** [prisma/migrate#507]
I have this schema already set up
``` 
model PaymentMethod {
  id                    Int     @default(autoincrement()) @id
  setupIntentId         String  @unique
  user                  User    @relation(fields: [userId], references: [id])
  userId                Int
  createdAt             DateTime @default(now())
}
``` 
And I renamed setupIntent and removed the unique flag. 
When I run the migration, I get the following error
``` 
Failure during a migration command: Connector error. (error: Error querying the database: Error querying the database: table "new_Paym
entMethod" already exists
   0: migration_core::api::ApplyMigration
           with migration_id="20200708232341-fixing-payment-method"
             at migration-engine/core/src/api.rs:77)
We successfully received the error report
To help us even more, please create an issue at https://github.com/prisma/prisma/issues/new
mentioning the report id 8707.
```

Any ideas?
Thanks

**Comment by tomhoule** [comment]
Thanks for reporting this! I assume this is on sqlite? Did you get warnings? If yes, can you reproduce the migration and paste the warnings here? Some of them should prevent you from running the migration soon.

**Comment by kurt-hardy** [comment]
This was SQL Lite. We wanted to use PG anyway so I ended up deleting the SQL lite file and moving over to postgres.

I have tried reverting back to recreate but I cant unfortunately. 
This was the error I got but I don't remember getting any warnings other than the data warning because I had data in the column to be rename. The logged out changes looked correct too (setup intent id would be renamed).

There's nothing specific I have to add to the command to tell prisma that I'm happy to lose the data in the columns is there?
Or would I have to make the renamed setupIntentId column nullable?

Thats not very helpful so feel free to close and if I can recreate, I'll zip up the changes so you can have a better look :-)
Thanks

**Comment by tomhoule** [comment]
This can be dealt with differently in the preview version of migrate (released in 2.13.0). We would consider this as a general case of a partially failed migration. Thanks for reporting this issue!
